commit 4a7c459e37d77b05f258001ccae99935fb660caf
Author: Mike Jerris <mike@jerris.com>
Date:   Wed Mar 15 15:02:04 2017 -0500

    FS-10074: [libtiff] stop using embedded libtiff.  Build now requires system libtiff

--- a/src/mod/endpoints/mod_gsmopen/Makefile.am
+++ b/src/mod/endpoints/mod_gsmopen/Makefile.am
@@ -1,10 +1,6 @@
 include $(top_srcdir)/build/modmake.rulesam
 MODNAME=mod_gsmopen
 
-TIFF_DIR=$(switch_srcdir)/libs/tiff-4.0.2
-TIFF_BUILDDIR=$(switch_builddir)/libs/tiff-4.0.2
-TIFF_LA=$(TIFF_BUILDDIR)/libtiff/libtiff.la
-
 SPANDSP_DIR=$(switch_srcdir)/libs/spandsp
 SPANDSP_BUILDDIR=$(switch_builddir)/libs/spandsp
 SPANDSP_LA=$(SPANDSP_BUILDDIR)/src/libspandsp.la
@@ -12,16 +8,12 @@ SPANDSP_LA=$(SPANDSP_BUILDDIR)/src/libsp
 mod_LTLIBRARIES = mod_gsmopen.la
 mod_gsmopen_la_SOURCES  = mod_gsmopen.cpp gsmopen_protocol.cpp 
 mod_gsmopen_la_CXXFLAGS = $(SWITCH_AM_CXXFLAGS)
-mod_gsmopen_la_CPPFLAGS = -I$(SPANDSP_DIR)/src -I$(TIFF_DIR)/libtiff -I$(SPANDSP_BUILDDIR)/src -I$(TIFF_BUILDDIR)/libtiff -I.
-mod_gsmopen_la_LIBADD   = $(switch_builddir)/libfreeswitch.la $(SPANDSP_LA) $(TIFF_LA)
+mod_gsmopen_la_CPPFLAGS = -I$(SPANDSP_DIR)/src -I$(SPANDSP_BUILDDIR)/src -I.
+mod_gsmopen_la_LIBADD   = $(switch_builddir)/libfreeswitch.la $(SPANDSP_LA)
 mod_gsmopen_la_LDFLAGS  = -avoid-version -module -no-undefined -lctb-0.16 -lgsmme
 
-BUILT_SOURCES = $(TIFF_LA) $(SPANDSP_LA)
-
-$(SPANDSP_LA): $(TIFF_LA) $(SPANDSP_DIR) $(SPANDSP_DIR)/.update
-	cd $(SPANDSP_BUILDDIR) && $(MAKE) CPPFLAGS="$(CPPFLAGS) -I$(TIFF_BUILDDIR)/libtiff -I$(TIFF_DIR)/libtiff" CFLAGS="$(CFLAGS)"
-	$(TOUCH_TARGET)
+BUILT_SOURCES = $(SPANDSP_LA)
 
-$(TIFF_LA): $(TIFF_DIR) $(TIFF_DIR)/.update
-	cd $(TIFF_BUILDDIR) && $(MAKE)
+$(SPANDSP_LA): $(SPANDSP_DIR) $(SPANDSP_DIR)/.update
+	cd $(SPANDSP_BUILDDIR) && $(MAKE) CPPFLAGS="$(CPPFLAGS)" CFLAGS="$(CFLAGS)"
 	$(TOUCH_TARGET)
